Documentation/ApiOverview/FormProtection/Index.rst

```diff
@@ -89,54 +72,14 @@ you can check that the form token is valid like this:
         // create a flash message for an invalid token
     }
 
-
-..  note::
-    The :php:`validateToken()` method invalidates the token with the token ID.
```

Comment:
Is this not true anymore? Why did you remove the note?

Reply:
Nope, the method does not exist anymore. Also - when looking at the code - the token is just validated from the input and the beuser session.

I will provide a follow-up for 11.5 for this part.

I tried to build an implementation for examples ext, but can't figure out an easy scenario. When creating a link with a be view helper (like for a form action) the token is already there, so no need to create it and validate on your own.